### ISSUE TITLE
latest release doesn't work anymore

### ISSUE THREAD

**Issue by runcom** [openshift/source-to-image#1094]
```
$ go install github.com/openshift/source-to-image/cmd/s2i@latest
go: github.com/openshift/source-to-image/cmd/s2i@latest (in github.com/openshift/source-to-image@v1.3.1):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```
go version on my fedora is 1.18, trying to use the main branch errors out on some btrfs deps

**Comment by coreydaley** [comment]
It looks like we need to cut a new release with the go 1.18 updates.

Can you open an issue for the btrfs issue as well with the details of the error you are getting?

**Comment by jacktar283** [comment]
/remove-lifecycle stale

Still experiencing the same issue with go 1.18.4 and s2i v1.3.1

**Comment by yselkowitz** [comment]
@coreydaley anything else needed to cut a new release?

**Comment by coreydaley** [comment]
I have built and released v1.3.2 which uses go1.18
https://github.com/openshift/source-to-image/releases/tag/v1.3.2
